### ISSUE TITLE
Restify 7.x is not supported

### ISSUE THREAD

**Issue by frbuceta** [tests-always-included/restify-router-magic#4]
```
assert.js:49
  throw new AssertionError(obj);
  ^

AssertionError [ERR_ASSERTION]: Method 'POST' already declared for route '/auth/forgotPassword'
    at Router._insert (MY_ROOT_DIRECTORY/node_modules/find-my-way/index.js:208:7)
    at Router._on (MY_ROOT_DIRECTORY/node_modules/find-my-way/index.js:135:8)
    at Router.on (MY_ROOT_DIRECTORY/node_modules/find-my-way/index.js:47:8)
    at RouterRegistryRadix.add (MY_ROOT_DIRECTORY/node_modules/restify/lib/routerRegistryRadix.js:42:21)
    at Router.mount (MY_ROOT_DIRECTORY/node_modules/restify/lib/router.js:167:20)
    at Server.serverMethod [as post] (MY_ROOT_DIRECTORY/node_modules/restify/lib/server.js:1476:33)
    at MY_ROOT_DIRECTORY/node_modules/restify-router-magic/lib/restify-router-magic.js:67:51
    at Array.forEach (<anonymous>)
    at MY_ROOT_DIRECTORY/node_modules/restify-router-magic/lib/restify-router-magic.js:48:27
    at loadRoutes (MY_ROOT_DIRECTORY/node_modules/restify-router-magic/lib/load-routes.js:83:9)
    at MY_ROOT_DIRECTORY/node_modules/restify-router-magic/lib/restify-router-magic.js:36:13
    at f (MY_ROOT_DIRECTORY/node_modules/once/once.js:25:25)
    at Glob.<anonymous> (MY_ROOT_DIRECTORY/node_modules/glob/glob.js:151:7)
    at Glob.emit (events.js:180:13)
    at Glob.emit (domain.js:422:20)
    at Glob._finish (MY_ROOT_DIRECTORY/node_modules/glob/glob.js:197:8)
    at done (MY_ROOT_DIRECTORY/node_modules/glob/glob.js:182:14)
    at Glob._processGlobStar2 (MY_ROOT_DIRECTORY/node_modules/glob/glob.js:637:12)
    at MY_ROOT_DIRECTORY/node_modules/glob/glob.js:626:10
    at RES (MY_ROOT_DIRECTORY/node_modules/inflight/inflight.js:31:16)
    at f (MY_ROOT_DIRECTORY/node_modules/once/once.js:25:25)
    at lstatcb_ (MY_ROOT_DIRECTORY/node_modules/glob/glob.js:519:7)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by fidian** [comment]
I'm using restify 7.1.1 and the example from the README, but I slightly changed `routes/index.js`:

```
module.exports = {
    get: (req, res, next) => {
        res.send("Hello world!");
        next();
    },
    post: (req, res, next) => {
        res.send("You did a POST");
        next();
    }
};
```

Using `curl localhost:8080` works as expected, as does `curl -X POST localhost:8080`. I suspect there's a problem in your files. Want to share `route/auth/forgotPassword.js` with me?

**Comment by frbuceta** [comment]
the files with '-' in the name, crash

For example:
```
index-f.js

module.exports = {
    get: (req, res, next) => {
        res.send("Hello world!");
        next();
    },
    post: (req, res, next) => {
        res.send("You did a POST");
        next();
    }
};
```

**Comment by fidian** [comment]
Thanks for supplying a way to reproduce this issue!

I've addressed this issue and added tests to make sure it doesn't happen again.

Released in 1.1.1.
